exercises/DI/exercise/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "angular2-webpack-build-DI-exercise",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "npm run clean && cross-env NODE_ENV=production webpack",
+    "dev": "cross-env NODE_ENV=development webpack-dev-server -d --inline --hot --progress --no-info",
+    "start": "npm run build && http-server ./dist"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "repository": {},
+  "devDependencies": {
+    "@angular/common": "2.1.0",
+    "@angular/compiler": "2.1.0",
+    "@angular/core": "2.1.0",
+    "@angular/platform-browser": "2.1.0",
+    "@angular/platform-browser-dynamic": "2.1.0",
+    "@types/core-js": "0.9.33",
+    "@types/node": "6.0.39",
+    "awesome-typescript-loader": "2.2.1",
+    "babel-core": "^6.17.0",
+    "babel-loader": "^6.2.5",
+    "babel-preset-es2015": "^6.16.0",
+    "basscss": "^8.0.1",
```

Comment:
same comment as David's from pull request #24 (David: I think we shouldn't be using any "advanced" css framework, it might confuse people specially since we are not explaining css in the training)